extensions/github/src/create-issue.tsx

```diff
@@ -123,7 +123,7 @@ export function IssueForm({ draftValues }: IssueFormProps) {
   const milestones = data?.repository?.milestones?.nodes;
 
   useEffect(() => {
-    setValue("title", "");
+    setValue("title", values.title);
```

Comment:
Isn't removing the line enough to fix it?

Reply:
You are right @thomaslombart, this has been fixed